src/renderer/page/settings/view.jsx

```diff
@@ -1,352 +1,322 @@
-              ))}
-            </FormField>
-
-            <FormRow
-              type="checkbox"
-              onChange={this.onAutomaticDarkModeChange.bind(this)}
-              defaultChecked={automaticDarkModeEnabled}
-              label={__('Automatic dark mode (9pm to 8am)')}
-            />
-          </div>
-        </section>
-
-        <section className="card">
-          <div className="card__content">
-            <h4>{__('Application Cache')}</h4>
-          </div>
-          <div className="card__content">
-            <p>
-              <Link
-                label={this.state.clearingCache ? __('Clearing') : __('Clear the cache')}
-                icon="icon-trash"
-                button="alt"
-                onClick={this.clearCache.bind(this)}
-                disabled={this.state.clearingCache}
+              </div>
+            </section>
+
+            <section className="card card--section">
+              <div className="card__title">{__('Share Diagnostic Data')}</div>
+              <div className="card__subtitle">{__('List what we are doing with the data here')}</div>
+              <div className="card__content">
+                <FormField
+                  type="checkbox"
+                  name="share_usage_data"
+                  onChange={this.onShareDataChange}
+                  checked={daemonSettings.share_usage_data}
+                  postfix={__(
+                    'Help make LBRY better by contributing diagnostic data about my usage'
+                  )}
+                />
+              </div>
+            </section>
+
+            <section className="card card--section">
+              <div className="card__title">{__('Theme')}</div>
+
+              <FormField
+                name="theme_select"
+                render={() => (
+                  <select
+                    name="theme_select"
+                    id="theme_select"
+                    onChange={this.onThemeChange}
+                    value={currentTheme}
+                    disabled={automaticDarkModeEnabled}
+                  >
+                    {themes.map(theme => (
+                      <option key={theme} value={theme}>
+                        {theme}
+                      </option>
+                    ))}
+                  </select>
+                )}
               />
-            </p>
-          </div>
-        </section>
+
+              <FormField
+                type="checkbox"
+                name="automatic_dark_mode"
+                onChange={this.onAutomaticDarkModeChange}
+                checked={automaticDarkModeEnabled}
+                postfix={__('Automatic dark mode (9pm to 8am)')}
+              />
+            </section>
+
+            <section className="card card--section">
+              <div className="card__title">{__('Application Cache')}</div>
+              <span className="card__subtitle">
+                {__('This will delete your subscriptions, ... other stuff')}
```

Comment:
I would like to add exactly what happens when you clear your cache. Other items to add?